tests_smoke/smoke_loop.py

```diff
@@ -0,0 +1,97 @@
+import os
+import subprocess
+import time
+from argparse import ArgumentParser
+
+from smoke.common import Attachment_type, Config, Notification_type  # type: ignore
+from smoke.test_admin_csv import test_admin_csv  # type: ignore
+from smoke.test_admin_one_off import test_admin_one_off  # type: ignore
+from smoke.test_api_bulk import test_api_bulk  # type: ignore
+from smoke.test_api_one_off import test_api_one_off  # type: ignore
+
+runCount = 0
+maxRuns = 0
+
+
+class bcolors:
+
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKCYAN = '\033[96m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+
+def init():
+
+    global maxRuns
+
+    parser = ArgumentParser()
+    parser.add_argument("-m", "--max-runs", dest="maxruns",
```

Comment:
you could add this documentation in either the Makefile and on top of this script so folks have an example of how to run it